src/js/core/directives/ui-grid-menu.js

```diff
@@ -294,7 +294,13 @@ function ($compile, $timeout, $window, $document, gridUtil, uiGridConstants, i18
                 $scope.$emit('hide-menu');
               } else {
                 // Maintain focus on the selected item
-                gridUtil.focus.bySelector(angular.element($event.target.parentElement), 'button[type=button]', true);
+                var correctParent = $event.target.parentElement;
+
+                if (angular.element($event.target)[0].nodeName === 'I') {
```

Comment:
Could you add a comment above the if statement explaining what is 'I' in this case? This is mainly because this line is not self explanatory and I don't want us to lose track as to what it was added for.

Reply:
Updated a while ago. Hope that's what you were looking for, and glad to be of help.